try:
    num = int(input("Enter a number: "))
    print("the number entered is: ", num)
except ValueError as ex:
    print("Exception occurred: ", ex)

# division


